source/Playnite/WebView/OffscreenWebView.cs

```diff
@@ -66,6 +69,13 @@ private void Browser_LoadingStateChanged(object sender, LoadingStateChangedEvent
             NavigationChanged?.Invoke(this, new EventArgs());
         }
 
+        private void Browser_JavascriptMessageReceived(object sender, JavascriptMessageReceivedEventArgs e)
+        {
+
+            JavascriptMessageReceived?.Invoke(this, new WebViewJavascriptMessageReceivedEventArgs { message = e.Message.ToString() });
+            NavigationChanged?.Invoke(this, new EventArgs());

```

Comment:
Why is NavigationChanged being called here? This should only JavascriptMessageReceived.

Reply:
This function shoud be a copy and paste of the WebView version, You're right.